etl/steps/data/garden/oecd/2023-05-19/co2_air_transport.meta.yml

```diff
@@ -78,8 +85,15 @@ tables:
         short_unit: lb
```

Comment:
I'm curious why we are using lb here - I would have thought kg would have been the more natural unit?